src/components/brand/brand.tsx

```diff
@@ -15,8 +15,9 @@ const Brand: React.FC<Props> = ({ height, width }: Props) => {
       width={`${width}px`}
       justifyContent="center"
       alignItems="center"
+      backgroundColor={'primary.500'}
     >
-      <Heading>Jalan Technologies</Heading>
+      <Heading color={'white'}>Better Software</Heading>
```

Comment:
Does this need to use theme? For example, I would expect that this to react to black/white theme. and if its light theme, white color will not work.